### PR TITLE
Features => Production

### PR DESCRIPTION
**Pull request by carlos-ssh**
I created these changes requested by the project.
- Set up a new project with webpack that is based on the webpack exercise you have already completed.- add webpack config file.
- Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder (<div> or <ul> element). The index.html file must be set as a template using the HTML Webpack Plugin.
- Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- On page load render the dynamically created list of tasks in the dedicated placeholder. The list should apear in order of the index values for each task.
- Create a style.css and set rules for the To Do List.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  {
+    description: "Go at the supermarket",
+    completed: true,
+    //index: 4,
+  },
+];
+
+const displayTodos = () => {
+  todos.forEach((item, index) => {
+    console.log(item);
```
Comment (bellom) [inline src/index.js]: ### JavaScript requirements

Let's remove `console.log`

image

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    const checkItem = item.completed ? 'checked' : '';
+    const lineTrought = item.completed ? 'trought-line' : '';
+    item.index = index;
+    console.log(item.description);
```
Comment (bellom) [inline src/index.js]: Let's remove `console.log`

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    const listElement =   
+     `<li>` +
+	`<input type="checkbox" class="checkbox" id="checkbox-${item.index}" ${checkItem}> ` +
+	  `<h4 class="text ${lineTrought} trought-line"> ${item.description} </h4>` +
+	  `<img src="https://img.icons8.com/ios-glyphs/30/000000/menu-2.png" alt="Vertical Menu">` +
+      `</li>`;
+    console.log(listElement);
```
Comment (bellom) [inline src/index.js]: Let's remove `console.log`

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @carlos-ssh  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well with the design, setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 

_Kindly check the comments under the review._


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (carlos-ssh) [reply src/index.js]: Thank you for review my project. I improved these requested changes.

File: src/index.js (hunk 2 above)
Comment (carlos-ssh) [reply src/index.js]: Thank you for review my project. I improved these requested changes.

File: src/index.js (hunk 3 above)
Comment (carlos-ssh) [reply src/index.js]: Thank you for review my project. I improved these requested changes.

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**